app/src/main/java/com/duckduckgo/app/survey/ui/SurveyActivity.kt

```diff
@@ -153,7 +153,7 @@ class SurveyActivity : DuckDuckGoActivity() {
         }
 
         override fun shouldInterceptRequest(view: WebView, request: WebResourceRequest): WebResourceResponse? {
-            if (request.isForMainFrame && request.url.host == "www.duckduckgo.com") {
+            if (request.url.host == "duckduckgo.com") {
```

Comment:
questions: why are we removing the `isForMainFrame` check? I assume we know it's going to be always `true`?

Reply:
No the opposite, it's false. Something in surveymonkey has changed and it's not loading our page as a mainframe request anymore so we need to drop that requirement. As soon as we start seeing requests for "duckduckgo.com" we know we have finished with the survey.